web/firefoxcom.js

```diff
@@ -338,14 +344,32 @@ class MLManager {
     this.altTextLearnMoreUrl = altTextLearnMoreUrl;
   }
 
+  async toggleService(name, enabled) {
+    if (name !== "altText") {
+      return;
+    }
+
+    if (enabled) {
+      await this.#loadAltTextEngine(false);
+    } else {
+      this.#enabled.delete(name);
```

Comment:
Should we not remove it from `this.#ready` as well, to avoid inconsistent state?